### ISSUE TITLE
Version 1.8.4 server pack - Incompatible FML modded server error

### ISSUE THREAD

**Issue by Nimso** [The-Nexus-Project/Limitless-3#80]
Version 1.8.4 server pack installed on a server.

When trying to connect to it, I am getting the Incompatible FML modded server error.
This error also shows when trying to connect to the Official TNP servers.

Client Version - 1.8.4 (Downloaded/installed via Curseforge app
Server Version - 1.8.4 (Downloaded from official Curseforge site (https://www.curseforge.com/minecraft/modpacks/tnp-limitless-3) (I also tried downloading the server pack from the Curseforge app and received the same results)

I also notice that the Client has 266 files in the mods folder, while the server has 245 files.

**Comment by Gbergz** [comment]
Don't look at the server in the multiplayer list. It'll always show that its "incompatible" while you can still join.
Did you even try to join before reporting this? 
Edit: You probably did. I've just tried new fresh instance & client. No problems. Have you modified anything?

Also,
Server files has less mods because client mods are removed there, no need for them.

Edit 2: And it seems that you're using the CurseForge app which could be the problem, sometimes it fails to download mods and this could be it. Try using MultiMC: https://multimc.org/#Download **OR** GDLauncher: https://gdevs.io/#downloadContainer 
And see how it goes.

**Comment by Nimso** [comment]
Thank you for your response.  I did try to join the server before writing this up, and could not connect. My wife and son could not connect either.  I tried loading 1.8.3a as well on the server and all 3 clients and got the same error, so I don't know what is going on.  I am using the Curse Launcher but I have been using this for quite some time (Back when it was Curse, and then when Twitch bought them out, and now when Overwolf bought it from Twitch and named it the CurseForge app)

Also, I have been running several different servers using the CurseForge app and have not had any problems with getting things set up.

I figured that the server/client might have different numbers of mods, but I was just trying to include as much info as I could think of that would help give an idea as to what was going wrong.

I just set up All The Mods 6 and it is working.  It is a shame, because I really wanted to play TNP Limitless 3 because of all of the quests/etc.  I will just have to use All The Mods 6 for now and try TNP L3 later on I guess.
